packages/perception/deps/20-librealsense.deps

```diff
@@ -78,11 +78,11 @@ build_from_src()
   cd librealsense
   mkdir -p build
   cd build
-  cmake ../
+  cmake "${TARGET_DIR}/librealsense"
   make -j8
   sudo make uninstall
   sudo make install
-  cd ../
+  cd "${TARGET_DIR}/librealsense"
   sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/
```

Comment:
line 85 -86 could be merged as below
` sudo cp "${TARGET_DIR}/librealsense/config/99-realsense-libusb.rules"  /etc/udev/rules.d/